CHANGELOG.md

```diff
@@ -1,4 +1,26 @@
 # Changelog
+Attention, this release contains critical security fixes. Please update as soon as possible. Additionally, we are
+encouraging users to change your Sunshine password, especially if you expose the web UI (i.e. port 47790 by default)
+to the internet, or have ever uploaded your logs with verbose output to a public resource.
+
+## [0.18.0] - 2023-01-25
+### Added
+- (Windows) Add support for Intel QuickSync
+- (Linux) Added aarch64 deb and rpm packages
+- (Windows) Add support for hybrid graphics systems
+- (Linux) Add support for streaming from Steam Deck Gaming Mode
+### Fixed
+- (Network) Refactor code for UPnP port forwarding
+- (Video) Enforce 10 FPS encoding frame rate minimum to improve static image quality
+- (Linux) deb and rpm packages are now specific to destination distro and version
+- (Docs) Add nvidia/nvenc preset migration guide
+- (Network) Performance optimizations
+- (Video) Fix HEVC hang
```

Comment:
I assume you're talking about the Intel QuickSync HEVC hang. That bug never actually shipped in any release, so not sure if we want to mention it here.